src/js/components/SchemaForm.js

```diff
@@ -84,6 +95,63 @@ class SchemaForm extends React.Component {
     return this.getDataTriple();
   }
 
+  handleRemoveRow(definition, prop, id) {
+    FormUtil.removePropID(definition, prop, id);
+    this.forceUpdate();
+  }
+
+  handleAddRow(prop, definition, newDefinition) {
+    let propID = Util.uniqueID(prop);
+    newDefinition = FormUtil.getMultipleFieldDefinition(
+      prop,
+      propID,
+      newDefinition
+    );
+    newDefinition.push(
+      this.getRemoveRowButton(definition, prop, propID)
+    );
+
+    // Default to prepending.
+    let lastIndex = -1;
+    definition.forEach(function (field, i) {
+      if (FormUtil.isFieldInstanceOfProp(prop, field)) {
+        lastIndex = i;
```

Comment:
Is it Ok to assume you'll always want the global last? If we have multiple groupings of this field, it will always append to the last grouping.